docs/source/guides/tools/SubtitleModding.rst

```diff
@@ -0,0 +1,56 @@
+Subtitle Modding
+===========
+
+Subtitles Overview
+-------------------------
+
+In Titanfall 2 Subtitles are stored in ``.dat`` files which are essentially just "Valve Close Caption Data" files.
+These files are located in the frontend VPK's.
+
+
+Prequisites
+-----------
+•	`VCCDSON <https://github.com/EM4Volts/vccdson>`__ 
+
+
+.. note::
+    VCCDSON is new and not tested with exotic characters from non latin languages.
+    If used for cyrillic languages or other languages with non latin symbols
+    it may ERROR or output non-valid .dat files.
+    Please report Errors on github as an issue.
+
+
+Editing Subtitles
+-----------------
+1. With a valid installation of Python 3 and VCCDSON just drag your desired subtitle ``.dat`` file on VCCDSON's ``main.py``.
+2. Open the newly created ``.json`` and edit the Subtitles to your liking.
+3. When done drag the ``.json`` on VCCDSON's ``main.py``.
+4. Your new subtitle ``.dat`` should now be created.
+
+    
+Commands
+^^^^^^^^^
+
+Subtitle strings can contain command blocks, recognizable by them being enclosed in < >
```

Comment:
```suggestion
Strings that can be inserted into to change the following text, they are enclosed with ``< >``
```

Reply:
sounds more confusing then it did before ngl